config.go

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2025 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dynamodb
+
+// Config contains shared config parameters, common to the source and
+// destination. If you don't need shared parameters you can entirely remove this
+// file.
+type Config struct {
+	// Table is the DynamoDB table name to pull data from.
```

Comment:
Well, it's not only to pull data from anymore, since we're adding a destination now 🙃